.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Please add the target folder here as well. I don't think you need the latest test results and artifacts in your version control.